src/Components/Task-editor.tsx

```diff
@@ -0,0 +1,63 @@
+import { inject, observer } from "mobx-react";
+import React from "react";
+import TaskItem from "../Models/TaskItem";
+import TaskStore from "../Stores/TaskStore";
+
+@inject('TaskStore')
+@observer
+export default class TaskEditor extends React.Component<{ taskStore: typeof TaskStore }> {
+    textTask: any;
+
+    addTodo = (arg0: { title: string; completed: boolean; }) => {
+        debugger;
+        let item: TaskItem = {            
+            title: arg0.title,
+            completed: arg0.completed
+        }
+        this.props.taskStore.addTask(item);
+    }
+
+    render() {
+        return (
+            <React.Fragment>
+                <div className="alert alert-primary">
+                    <div className="d-inline col-4">
+                        Total items: &nbsp;
+                        <span className="badge badge-info">{this.props.taskStore.getTaskCount.total}</span>                       
+                    </div>
+                    <div className="d-inline col-4">
+                        Finished items: &nbsp;
+                        <span className="badge badge-info">{this.props.taskStore.getTaskCount.completed}</span>                        
+                    </div>
+                    <div className="d-inline col-4">
+                        Unfinished items: &nbsp;
+                        <span className="badge badge-info">{this.props.taskStore.getTaskCount.notCompleted}</span>                       
+                    </div>
+                </div>
+                <div className="form-group">
+
+                    <input type="text" className="form-control" ref={e => this.textTask = e} placeholder="Search..." />
```

Comment:
hardcoded place holder , this should be `add todo ` i guess. Better not to put pr with these.
Better to introduce localization or  since this is a tryout project , add these into const file

Reply:
Stil in progress

Reply:
I have used config-name json file to replace name on html elements
I will do some localization without library. But still in progress